app/models/user.rb

```diff
@@ -1,9 +1,9 @@
 class User < ApplicationRecord
-  has_many :posts, foreign_key: :user_id
-  has_many :comments, foreign_key: :user_id
-  has_many :likes, foreign_key: :user_id
+  has_many :posts, foreign_key: 'user_id'
+  has_many :comments, foreign_key: 'user_id'
+  has_many :likes, foreign_key: 'user_id'
```

Comment:
- [x] The foreign keys should be to `author_id` and not `user_id`. Kindly fix this. See suggestion below:

```suggestion
 has_many :posts, foreign_key: 'author_id'
has_many :comments, foreign_key: 'author_id'
has_many :likes, foreign_key: 'author_id'
```